lib/src/collection/lru_map.dart

```diff
@@ -195,10 +185,12 @@ class LinkedLruHashMap<K, V> implements LruMap<K, V> {
     return entry.value;
   }
 
-  /// Get the value for a [key] in the [Map]. The [key] will be promoted to the
-  /// 'Most Recently Used' position. *NOTE*: Calling [] inside an iteration
-  /// over keys/values is currently unsupported; use [keys] or [values] if you
-  /// need information about entries without modifying their position.
+  /// Get the value for a [key] in the [Map].
+  /// The [key] will be promoted to the 'Most Recently Used' position.
+  ///
+  /// *NOTE*: Calling [] inside an iteration over keys/values is currently
```

Comment:
`[]` should be between backticks otherwise they are not rendered. (See https://www.dartdocs.org/documentation/quiver/0.28.0/quiver.collection/LinkedLruHashMap/operator_get.html)

Reply:
Haha great catch!